ansible/roles/tldraw-client-core/templates/tldraw-client-ingress.yml.j2

```diff
@@ -0,0 +1,43 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ NAMESPACE }}-tldraw-client-ingress
+  namespace: {{ NAMESPACE }}
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-redirect: "{{ TLS_ENABLED | default("false") }}"
+    nginx.ingress.kubernetes.io/auth-type: basic
+    nginx.ingress.kubernetes.io/auth-secret: shd-basic-auth-secret
```

Comment:
Remove this Line to remove the HTTP-Basic Auth request.
Please lock if you do copy&past from other things like Ingresses what you copy.

Reply:
Okay, it's more then one line who must be removed